### ISSUE TITLE
Resource picker preview should be on the right, not left

### ISSUE THREAD

**Issue by BretJohnson** [xamarin/Xamarin.PropertyEditing#221]
Was there a reason we put the preview window on the left?  See screenshot below.

The current Android resource picker, on both VSWin and VSMac, has it on the right as does Android Studio.  Arguably, that's more logical, since the preview is a consequence of the current selection. And it's also more consistent with current UI.

I'm suggesting we just flip it, unless @vancura disagrees.

![image](https://user-images.githubusercontent.com/245892/36519818-9ecf7150-175b-11e8-8255-2e163c5fe9a9.png)

Current Windows UI, for comparison:

![image](https://user-images.githubusercontent.com/245892/36519887-fb541b56-175b-11e8-9d32-86bd11fed926.png)

**Comment by vancura** [comment]
Yes, there is a couple of reasons.

[Resource Selector on Mac](https://github.com/xamarin/Xamarin.PropertyEditing/issues/131) uses both sides of the preview pane, based on where the Property Pad is displayed in the IDE. If it's right-hand side (default in VSMac, VS, and Inspector), the Preview pane is displayed on the left side – because the left side is above the middle area of the IDE, and there's a big chance we can't move the popover is already close to the right edge of the screen. The table is the primary target, not the Preview, and it needs to be close to the Property Pad itself. When the Property Pad is on the left side, we reverse the polarity, so the Preview pane is on the right side, above the IDE center.

[On Windows](https://github.com/xamarin/Xamarin.PropertyEditing/issues/130) there's nothing like popovers, or at least nothing commonly used – we display the Resource Editor in a regular window. But there's a good reason why the Preview pane is left aligned: buttons. As the usual brain gives more importance on the right side of the world, buttons need to be on edge. Take a look at your screenshots. Buttons are in a completely different level than the Preview Pane, below everything. There, it doesn't matter. But we have buttons relative to the table, so it makes sense.

Also, see the discussion in linked tickets above.

**Comment by vancura** [comment]
TL;DR: If the Preview Pane is right aligned, buttons would be CENTERED in the window. Not good.

**Comment by CartBlanche** [comment]
@vancura I don't think it makes sense for the preview pane to take up the whole right or left side, either on Mac or Windows. IMHO it should be the same height as the listview/tableview where the resources are, that way it doesn't affect the OK/Cancel button alignments. After all the selected items in the listview/tableview are related to the preview, so why not have them visually at the same height, but left or right aligned? 
For example in the Windows UI example above that @BretJohnson showed, if that preview was on the left, the OK/Cancel would still be on the right edge, where it is in the original screen shot. 
Lastly what do we gain by having the preview take up the whole right/left side? It is quite a large preview area anyway for colours/bitmaps and the like.
Am I missing something?

Unrelated, I know we are supposed to stick to VS guidelines on Windows, but those pop up windows aren't nice. Couldn't we have the equivalent Callout/Popup Over on the Windows side, thus doing away with the OK/Cancel buttons?? At the last Evolve didn't we have a speaker talk about how buttons are evil :p . Maybe liaise with the VS team and suggest there is a "better" way to do that UX.

**Comment by vancura** [comment]
I don't see any issue with Preview Pane taking full height.

On Mac, the other pane has only the selector footer, and it needs to stick to the table. There's nothing else, so no reason not to have it full-height.

![](https://user-images.githubusercontent.com/4982/35865351-8f8082ec-0b54-11e8-9151-451512b25c44.png)

Here's how right-aligned Preview would look on Windows. The buttons are meh, as you can see.

<img width="886" alt="35797141-10a3c4de-0a5f-11e8-99b5-443705bad4a3" src="https://user-images.githubusercontent.com/4982/36541137-396b36b4-17dd-11e8-8297-87bfa668c694.png">

For the record, here's the same thing with the full-width footer. IMHO it's too busy.

<img width="886" alt="35797141-10a3c4de-0a5f-11e8-99b5-443705bad4a3" src="https://user-images.githubusercontent.com/4982/36541246-9b7fe08e-17dd-11e8-891b-e52fc11f4dad.png">

> Unrelated, I know we are supposed to stick to VS guidelines on Windows, but those pop up windows aren't nice. Couldn't we have the equivalent Callout/Popup Over on the Windows side, thus doing away with the OK/Cancel buttons?

I didn't know we have something like that in VS. How does it look?

**Comment by CartBlanche** [comment]
> I didn't know we have something like that in VS. How does it look?
Windows doesn't, but maybe we know some good xaml developers who could write one :D. Similar idea to this SO - https://stackoverflow.com/questions/11446250/how-to-style-wpf-tooltip-like-a-speech-bubble

**Comment by BretJohnson** [comment]
@vancura Your last image seemed ok to me (minus the Show Preview toggle). And FYI here's the Android Studio version.
But I'm also fine with it being on the left - it's not that big of deal.

![image](https://user-images.githubusercontent.com/245892/36543839-f24ec6b2-17b2-11e8-9d98-35ccc922b9bd.png)
